doc/userguide/partials/options.rst

```diff
@@ -16,10 +16,22 @@
 
    Test configuration.
 
+.. _cmdline-option-v:
+
 .. option:: -v
 
-   The -v option enables more verbosity of Suricata's output. Supply
-   multiple times for more verbosity.
+   Increase the verbosity of the Suricata application logging by
+   increasing the log level from the default. This option can be
+   passed multiple times to further increase the verbosity.
+
+   - -v: INFO
+   - -vv: PERF
+   - -vvv: CONFIG
+   - -vvvv: DEBUG
```

Comment:
This took me a few times to understand and perhaps a slight modification  would be something like:

```
Here are the logging level equivalents when specifying the verbose option:
- -v changes the log level to INFO
- -vv changes the log level to PERF
...
```

Reply:
Agreed on the wording, I will fix that.

As for the level used at runtime, no. Per the issue the discussion was having -v occurrences max to fixed levels based on what is the current hardcoded default of notice.